emu/src/cpu/arm/operations.rs

```diff
@@ -841,6 +836,293 @@ impl Arm7tdmi {
 
         self.flush_pipeline();
     }
+
+    #[allow(clippy::too_many_arguments)]
+    pub fn coprocessor_data_transfer(
+        &mut self,
+        indexing: Indexing,
+        offsetting: Offsetting,
+        _transfer_length: bool,
+        _write_back: bool,
+        _load_store: LoadStoreKind,
+        rn: u32,
+        _crd: u32,
+        _cp_number: u32,
+        offset: u32,
+    ) -> Option<u32> {
+        let mut _address = self.registers.register_at(rn.try_into().unwrap());
+        let effective = match offsetting {
+            Offsetting::Down => _address.wrapping_sub(offset),
+            Offsetting::Up => _address.wrapping_add(offset),
+        };
+
+        let _address = match indexing {
+            Indexing::Pre => effective,
+            Indexing::Post => _address,
+        };
+
+        // TODO: take a look if we need to finish this for real.
+        todo!("finish this");
+        // Some(SIZE_OF_ARM_INSTRUCTION)
+    }
+
+    pub fn multiply(&mut self, mul_variant: ArmModeMultiplyVariant, op_code: ArmModeOpcode) {
+        use ArmModeMultiplyVariant::*;
+
+        match mul_variant {
+            // Unsiged multiply (32-bit by 32-bit, bottom 32-bit result).
+            Mul => self.mul(op_code),
+            // Unsiged multiply-accumulate (32-bit by 32-bit, bottom 32-bit accumulate and result).
+            Mla => self.mla(op_code),
+        }
+    }
```

Comment:
I think we removed this right? @AlessioC31

Reply:
Yep, [GBA doesn't have a coprocessor](https://github.com/RIP-Comm/clementine/commit/7c895e703dd1611e65bf46acd20a7ab33e8a2f88)

Reply:
Looking good now? @guerinoni @AlessioC31